### PR TITLE
Update free-programming-books-ta.md

### PR DESCRIPTION
**Pull request by 0xParcival**
Add 11th std books pdf

## What does this PR do?

      Add resource

## For resources
         
   [Tamil Nadu Education ](https://www.tn.gov.in/department/28)

### Description

          it explain C++ ,Fundamental of Programming ,Computer Hardware , Html ,CSS,Javascript

### Why is this valuable (or not)?

             It Valuable for Beginner 

### How do we know it's really free?
         its Issued by Government of Tamil Nadu 
### For book lists, is it a book? For course lists, is it a course? etc.
                 It Book 

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by 0xParcival — COMMENTED** [review]
added 11th computer book in tamil

**Comment by 0xParcival** [comment]
Added 11th std book in tamil lang

File: books/free-programming-books-ta.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ## Index
 
+* [11th Std Computer Science Book](#11th-Std-computer-book)
```
[the remark this answers is not in the document]
Comment (0xParcival) [reply books/free-programming-books-ta.md:3]: ok i try to more books

**Comment by 0xParcival** [comment]
Extra book added

**Comment by 0xParcival** [comment]
> can't list a book if it's only available in your country.

I can find pdf in other website but there are in Google Drive.

**Comment by eshellman** [comment]
as I explained, we can't list this books - they are not free.

**Pull request closed without merging**